Firestore/Example/Tests/Util/FSTHelpers.mm

```diff
@@ -277,9 +281,10 @@ NSComparator FSTTestDocComparator(const absl::string_view fieldPath) {
   NSMutableArray<FSTFieldTransform *> *fieldTransforms = [NSMutableArray array];
   for (NSString *field in serverTimestampFields) {
     const FieldPath fieldPath = testutil::Field(util::MakeStringView(field));
-    id<FSTTransformOperation> transformOp = [FSTServerTimestampTransform serverTimestampTransform];
+    std::unique_ptr<ServerTimestampTransform> transformOp =
```

Comment:
Nit: this looks like a good place to use `auto`, because `make_unique` makes the variable type pretty clear.

Reply:
done